plugins/additional/additional.cpp

```diff
@@ -416,19 +416,54 @@ class HtmlWindowComponent : public ComponentBase
 
 
 
-class ToggleButtonComponent : public ComponentBase, public wxEvtHandler
+class ToggleButtonComponent : public ComponentBase
 {
 public:
 	wxObject* Create(IObject* obj, wxObject* parent) override {
-		wxToggleButton* window = new wxToggleButton((wxWindow *)parent, wxID_ANY,
-			obj->GetPropertyAsString(_("label")),
+		wxString label = obj->GetPropertyAsString( _("label") );
+		wxToggleButton* button = new wxToggleButton((wxWindow *)parent, wxID_ANY,
+			label,
 			obj->GetPropertyAsPoint(_("pos")),
 			obj->GetPropertyAsSize(_("size")),
-			obj->GetPropertyAsInteger(_("window_style")));
+			obj->GetPropertyAsInteger(_("style")) | obj->GetPropertyAsInteger(_("window_style")));
 
-		window->SetValue( ( obj->GetPropertyAsInteger(_("value")) != 0 ) );
-		window->Connect( wxEVT_COMMAND_TOGGLEBUTTON_CLICKED, wxCommandEventHandler( ToggleButtonComponent::OnToggle ), NULL, this );
```

Comment:
This removal breaks the updating of the value if the button gets clicked in the preview, i see no reason why this feature should be removed